Allow disabling building examples, as they are not installed anyways.

--- CMakeLists.txt.orig	2019-06-23 11:56:17 UTC
+++ CMakeLists.txt
@@ -53,7 +53,7 @@ if (NOT CMAKE_CROSSCOMPILING)
 	add_subdirectory(data)
 endif()
 
-if(TARGET Qt5::Widgets)
+if((TARGET Qt5::Widgets) AND (ENABLE_EXAMPLES))
     add_subdirectory(examples)
 endif()
 
